glass/src/lib/native/cpp/other/Log.cpp

```diff
@@ -65,6 +69,12 @@ void LogView::Display() {
     if (ImGui::Selectable("Clear")) {
       m_data->Clear();
     }
+    if (ImGui::Selectable("Copy to Clipboard")) {
```

Comment:
I'm thinking we should disable the selectable when the buffer is empty, since it's a no-op?  To do this, move the GetBuffer line above this one, then change the call to `ImGui::Selectable("Copy to Clipboard", false, buf.empty() ? ImGuiSelectableFlags_Disabled : 0)`

Reply:
We aren't disabling the `Clear` option when the buffer is empty, I don't see why the `Copy to Clipboard` option should be. Is this something we want to do?

Reply:
Well, the difference is that the user might expect it to copy something, then is confused when they paste?  We could disable both when the buffer is empty.

Reply:
What's the consensus?

Reply:
I think it makes sense to only disable "Copy" when the buffer is empty. "Clear" is technically still a valid operation even when the buffer is empty (visually, there is still nothing displayed).